StudentManagementSystemProject/src/main/java/com/zemoso/controller/LoginController.java

```diff
@@ -0,0 +1,25 @@
+package com.zemoso.controller;
+
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class LoginController {
+
+    @GetMapping("/showMyLoginPage")
+    public String showMyLoginPage() {
+
```

Comment:
Implement Logger logger = LoggerFactory.getLogger(className.class); from @Slf4j 
logger.info("On Login Page"); // customise these messages in controllers where ever required

Reply:
Okay